jenkins/main.groovy

```diff
@@ -14,6 +14,7 @@ def prStatusContext = 'jenkins/main'
 if (isPullRequest) {
     echo 'Setting Github PR status'
     githubNotify context: prStatusContext, status: 'PENDING', description: 'Build started'
+    notifyBuild('STARTED')
```

Comment:
We actually want to send notifications on non-PR builds too, so let's move this instance of `notifyBuild` just after line 18 so it's outside the conditional checking if it's a PR